5_databases/week1/Homework.sql

```diff
@@ -0,0 +1,58 @@
+-- Find out how many tasks are in the task table
+Select COUNT (*) 
+from task;
+
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT 
+(SELECT  COUNT (*) from task)-(SELECT COUNT( due_date) from task);
+
+
+-- Find all the tasks that are marked as done
+Select task.title,task.status_id
```

Comment:
Try to run this, it returns every task 3 times which we dont have. Think more simple:
```
SELECT *
FROM task
WHERE status_id = 3;
```